jerry-core/api/jerry.c

```diff
@@ -1070,7 +1070,7 @@ jerry_create_boolean (bool value) /**< bool value from which a jerry_value_t wil
 {
   jerry_assert_api_available ();
 
-  value = jerry_get_arg_value (value);
+  value = jerry_get_arg_value (value) ? true : false;
```

Comment:
I think the correct fix is to remove only the original line.
`jerry_get_arg_value (jerry_value_t value)`, but it is called with a bool and not a `jerry_value_t`.

Reply:
Good catch, I think this is incorrect. Probably should be removed.